docs/user/openstack/install_upi.md

```diff
@@ -617,34 +617,16 @@ In this guide, we will assume the file is at the following URL:
 
 https://static.example.com/bootstrap.ign
 
-##### Example 1: Swift
+##### Example: Glance image service
```

Comment:
Please also update the Table of Contents. I think we could remove the example links from it.